cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -14,14 +21,43 @@
     pytest.mark.tier(2)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
+
+def get_random_string(length):
+    valid_chars = digits + ascii_letters + ' !@#$%^&*()'
+    out = ''.join([choice(valid_chars) for _ in xrange(length)])
+    return re.sub('\s+', ' ', out)
+
+
 ATTRIBUTES_DATASET = [
     CustomAttribute('exp_date', '2017-01-02', 'Date'),
     CustomAttribute('sales_force_acount', 'ADF231VRWQ1', None),
-    CustomAttribute('expected_num_of_nodes', '2', None),
+    CustomAttribute('expected_num_of_nodes', '2', None)
 ]
 VALUE_UPDATES = ['2018-07-12', 'ADF231VRWQ1', '1']
 
 
+def setup_dataset(provider, verify_exists=False):
```

Comment:
Still reading the new test functions, but I'm skeptical of this method name.

I'm thinking it would be better to implement the setup as a fixture, and create a separate method to delete DATASET attributes (verify_exists=False mode)